Accessories/ExpansionBoard_uInverter/REV20210506A/Prj_uinv-rev-a.csv

```diff
@@ -0,0 +1,21 @@
+Supplier Part Number 1,Quantity,Comment,Description,Designator,Footprint,LibRef
+
+"490-10486-1-ND","9","2.2UF","CAP CER 2.2UF 35V X5R 0603","C1, C2, C3, C4, C5, C6, C9, C10, C11","0603_CAP","GRM188R6YA225MA12D"
+"P19719CT-ND","2","1000u","Cap Aluminum 1000uF 35V 20% (12.5 X 25mm) Radial 5mm 3190mA 7000 hr 105C Ammo Pack","C7, C8","CAP_PAN_12.5X25_PAN","EEU-FM1V102B"
+"CP-037AH-ND","1","PJ-037AH","CONN PWR JACK 2X5.5MM SOLDER","CONN1","CUI_PJ-037AH","PJ-037AH"
+"609-2802-ND","1","ICD15S13E6GV00LF","Conn High Density D-Sub SKT 15 POS 2.28mm Solder RA Thru-Hole 15 Terminal 1 Port","CONN2","ICD15S13E6GV00LF","ICD15S13E6GV00LF"
+"RJSAE-5385-02-ND","1","RJSAE-5385-02","CONN MOD JACK 8P8C R/A SHIELDED","CONN3","CONN_RJSAE538502_AMP","RJSAE-5385-02"
+"HS380-ND","3","530002B02500G","Heat Sink Passive TO-220 Radial Thru-Hole 2.6C/W Black Anodized","HS1, HS2, HS3","530002","530002B02500G"
+"277-1667-ND","7","1935161","Conn Terminal Blocks 2 POS 5mm Solder ST Thru-Hole 16A","J1, J2, J3, J4, J5, J6, J7","PHOENIX_1935161","1935161"
+"553-3011-1-ND","3","100u","FIXED IND 100UH 5.3A 118 MOHM","L1, L2, L3","FP-PA4344_104NLT-MFG","PA4344.104NLT"
+"","10","MOUNT_HOLE, TitleBlockWEMPEC","MOUNTING HOLE, [NoValue]","MH1, MH2, MH3, MH4, MH5, TitleBlockWEMPEC1, TitleBlockWEMPEC2_U_Half_bridge_load1, TitleBlockWEMPEC2_U_Half_bridge_load2, TitleBlockWEMPEC2_U_Half_bridge_load3, TitleBlockWEMPEC3","MOUNTING_HOLE, [NoValue]","MOUNT_HOLE, TitleBlockWEMPEC"
```

Comment:
In altium, change the type of the `MOUNTING_HOLE` and `TitleBlock` symbols to be either `Graphical` or `Mechanical` to not have them automatically included in the BOM.

Please make this change in altium, then regenerate the BOM file to ensure it is clean.

Reply:
Changes made in Altium file.